Exercise_D.js

```diff
@@ -15,6 +15,26 @@ Use a combination of conditionals and loops to create your answer for this quest
 
 function multiplyNumber(number) {
   // CODE HERE
-}
+  if (typeof number !=="number"){
+    return "ERROR";
+  }
+  if(number <1){
+    return "ERROR";
+  }
+
+  if(number <= 1000000){

```

Comment:
you will have to use while loop here as the question states "multiply the number by 10 however many times it takes 
 * to get a number that is greater than or equal to 1000000"
 
For example:
```
while (number < 1000000) {
     number *= 10;
}
return number
```